src/lib/core/public-api.ts

```diff
@@ -18,5 +18,8 @@ export * from './label/label-options';
 export * from './ripple/index';
 export * from './selection/index';
 
-// TODO: don't have this
-export * from './testing/month-constants';
+/**
+ * @deprecated Use month constants in `src/cdk/testing` instead.
```

Comment:
We actually don't expose these publicly, so people can't use them instead. They should just create their own month constants for their application